hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -361,7 +366,7 @@ protected String defaultWarehouseLocation(TableIdentifier tableIdentifier) {
     return meta;
   }
 
-  public  Database convertToDatabase(Namespace namespace, Map<String, String> meta) {
+  protected Database convertToDatabase(Namespace namespace, Map<String, String> meta) {
```

Comment:
Why is this `protected`?

Reply:
Because this method only used in hive, so I think protected is better.

Reply:
Should it be private? If it is used in tests, we would normally make it package-private.

Reply:
OK